package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "node-db-challenge",
+    "version": "1.0.0",
+    "main": "index.js",
+    "scripts": {
+        "server": "nodemon index.js",
+        "knex": "knex"
```

Comment:
you can total make a "latest" script if you were to want to get super lazy....LOL. 

```
"scripts" : {
        "server" : "nodemon index.js",
        "latest" : "knex migrate:latest",
        "back" :  "knex migrate:rollback"
}
```

Reply:
You would use yarn to use the script...so `yarn latest` would run the script under latest....etc.